lumicks/pylake/kymotracker/kymotracker.py

```diff
@@ -64,20 +65,20 @@ def track_greedy(
     together using a greedy forward search analogous to [2]. This in contrast with the linking
     algorithm in [1] which uses a graph based optimization approach.
 
-    The linking step traverses the kymograph, tracing lines starting from each frame. It starts with
-    the highest line and proceeds to lines with lower signal intensity. For every point along the
-    line, the algorithm makes a prediction for where the particle will be in the next frame. Points
-    are considered candidates for line membership when they fall within a cone parameterized by a
+    The linking step traverses the kymograph, tracing tracks starting from each frame. It starts with
+    the highest intensity track and proceeds to tracjs with lower signal intensity. For every point along the
+    track, the algorithm makes a prediction for where the particle will be in the next frame. Points
+    are considered candidates for track membership when they fall within a cone parameterized by a
     sigma and diffusion constant. The candidate point closest to the prediction is chosen and
-    connected to the line. When no more candidates are available the line is terminated.
+    connected to the track. When no more candidates are available the track is terminated.
 
     Parameters
     ----------
     kymograph : lumicks.pylake.Kymo
-        Kymograph.
-    channel : str
-        Kymograph channel.
-    line_width : float or None
+        The kymograph to track.
+    channel : {'red', 'green', 'blue'}
+        Color channel to track.
+    track_width : float
```

Comment:
or `None` ?

You explicitly state in the string describing it that it's an option.